crates/astria-sequencer-relayer/src/relayer/write/conversion.rs

```diff
@@ -159,6 +180,8 @@ pub(super) enum TryIntoPayloadError {
          could not be constructed"
     )]
     NoSequencerNamespacePresent,
+    #[error("the payload is too large")]
+    PayloadSize,
```

Comment:
This could give the impression that there is a defined and reasonable upper limit somewhere, leaving readers guessing:
```suggestion
    #[error("the payload size exceeded u64::MAX bytes")]
    PayloadSizeOverflowed,
```